region/wreckedship/main.json

```diff
@@ -2225,6 +2167,65 @@
             {
               "id": 1,
               "strats": [
+                {
+                  "name": "Leave with Runway",
+                  "notable": false,
+                  "requires": [],
+                  "exitCondition": {
+                    "leaveWithRunway": {
+                      "length": 2,
+                      "openEnd": 1
+                    }
+                  }
+                },
+                {
+                  "name": "Leave Shinecharged, Second Closest Runway",
```

Comment:
Strange that these `canLeaveCharged` didn't have any requirements on killing the kihunters before.

Reply:
This does seem like a problem. Fortunately, the enemies won't appear until Phantoon is defeated, so it will be very unlikely to matter. It also looks like you can farm energy from the left or right door atomic spawns (assuming it's not heated), then once full, easily kill a Kihunter while taking a hit, then farm the rest of the room to survive. Coming in from the bottom door is less forgiving though, if you enter with very low energy.

I'll put a redo of this room in my to-do list.